<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fmCh</class>
 <widget class="QDialog" name="fmCh">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/icons/icons/edit-icon.png</normaloff>:/icons/icons/edit-icon.png</iconset>
  </property>
  <property name="whatsThis">
   <string notr="true"/>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="lbName">
       <property name="text">
        <string>Имя</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbFamily">
       <property name="text">
        <string>Фамилия</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="leFamily"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lbPhone">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string>Телефон</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lePhone"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lbMail">
       <property name="text">
        <string>e_mail</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="leMail"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="lbNotes">
       <property name="text">
        <string>Заметки</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="leNote"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="leName">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>id</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="leId">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbOk">
       <property name="text">
        <string>ОК</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCansel">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leName</tabstop>
  <tabstop>leFamily</tabstop>
  <tabstop>lePhone</tabstop>
  <tabstop>leMail</tabstop>
  <tabstop>leNote</tabstop>
  <tabstop>pbOk</tabstop>
  <tabstop>pbCansel</tabstop>
  <tabstop>leId</tabstop>
 </tabstops>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pbCansel</sender>
   <signal>clicked()</signal>
   <receiver>fmCh</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
